test/NuGet.Core.Tests/NuGet.Frameworks.Test/NuGetFrameworkParseComponentsTest.cs

```diff
@@ -0,0 +1,127 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using NuGet.Frameworks;
+using Xunit;
+
+namespace NuGet.Test
+{
+    public class NuGetFrameworkParseComponentsTests
```

Comment:
All of these tests are passing in values of `tfi` that are not the same as what we will get from the .NET SDK, so there's a risk we're not testing the right thing. Plus, the .NET SDK will prefix all TFV values with `v`, which none of these tests are testing.